desktop/lib/messagebundles/default.po

```diff
@@ -4269,6 +4269,25 @@ msgstr ""
 msgid "Enable Distributed Hash Table (DHT)"
 msgstr ""
 
+#: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/options/panes/TorrentConnectionPaneItem.java:42
```

Comment:
Yeah, I was afraid of that. This direct path was suspicious. I asked him on slack about that, still waiting on a response.